CHANGELOG.md

```diff
@@ -16,12 +16,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * The MQTT settings validation handler now receives the already updated new settings
   and the old settings (instead of the not updated current settings and the new settings).
 * The MQTT client has been split into its own `miniconf_mqtt` crate.
+* The attribute syntax has changed from `#[tree(depth(1))]` to `#[tree(depth=1)]`.
```

Comment:
Need to add a note somewhere that `handled_update` was removed from the MQTT client. Do we want a separate CHANGELOG for that?

Reply:
Same as https://github.com/quartiq/miniconf/pull/198#discussion_r1565416042